### ISSUE TITLE
NPV giving wrong results

### ISSUE THREAD

**Issue by ajinkya976** [ismaelJimenez/finance#1]
Hi,

For below calculations NPV is giving wrong results.

`print(Finance.npv(rate: 0.05, values: <num>[1500, 2500, 3500, 4500, 6000]));`

**Answer through above code is : 15879.03959769849**

**But the correct answer is : 15122.89**

![npv calc](https://user-images.githubusercontent.com/54501149/99139250-0c977a80-265d-11eb-89a5-a12ba43c8b02.jpg)

**Comment by bkbonner** [comment]
Yep, you're correct.   It's because the author was using index for the time period (which is zero based in his code), but needs to have 1 added to correctly calculate the NPV.

I added a pull request to add null safety and to fix the NPV calculation and tests
